include/gsl/span

```diff
@@ -70,9 +71,9 @@ namespace gsl
 {
 
 // [views.constants], constants
-constexpr const std::ptrdiff_t dynamic_extent = -1;
+constexpr const std::size_t dynamic_extent = static_cast<std::size_t>(-1);
```

Comment:
Pre-existing: `constexpr const` has always been redundant on scalar variables like this one. You can simply say `constexpr`.